src/index.html

```diff
@@ -1,14 +1,60 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <!--Fonts-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <!--Favicon-->
+    <link rel="shortcut icon"
+          href="../src/images/logo.png"
+          ype="image/x-icon"
+    >
+    <!--Style-->
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home">
+          <img src="./images/logo.png" alt="Logo MOYO">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__menu">
+          <li class="nav__list">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__list">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__list">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__list">
+            <a href="#laptop"
+               class="nav__link"
+               data-qa="hover">Laptops & Computers</a>
```

Comment:
Need to check and fix it everywhere
<img width="1072" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/89244149-060b-41ff-b8fb-d21ea77c38a2">